content/courses/program-security/pda-sharing.md

```diff
@@ -63,31 +63,29 @@ pub struct WithdrawTokens<'info> {
     pool: Account<'info, TokenPool>,
     vault: Account<'info, TokenAccount>,
     withdraw_destination: Account<'info, TokenAccount>,
-    /// CHECK: This is the PDA that signs for the transfer
+    /// CHECK: PDA
```

Comment:
`/// CHECK` is supposed to be a sentence explaining why you're not checking the account.